perfmetrics/scripts/continuous_test/ml_tests/tf/continuous.cfg

```diff
@@ -0,0 +1,2 @@
+build_file: "gcsfuse/perfmetrics/scripts/continuous_test/ml_tests/tf/build.sh"
+timeout_mins: 5700
```

Comment:
Can we use 20160 minutes (14 days) here similar to [pytorch kokoro job](https://github.com/GoogleCloudPlatform/gcsfuse/pull/871/files) ?

Reply:
Yes. Done